code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -204,7 +204,7 @@
 
 /obj/item/reagent_containers/food/drinks/holyoil
 	name = "flask of zelus oil"
-	desc = "A brass flask of Zelus oil, a viscous fluid with a scenting of brass - this flask may be sipped or thrown."
+	desc = "A brass flask of Zelus oil, a viscous fluid with a scenting of brass. Can be thrown to deal damage from afar."
```

Comment:
```suggestion
	desc = "A brass flask of Zelus oil, a viscous fluid scenting of brass. Can be thrown to deal damage from afar."
```

Anyway, you need to point out the fact it's a single use as a throwing weapon, like most other drink containers.

Reply:
Thats in the cult paper thing they get